runtime/url.cpp

```diff
@@ -4,80 +4,17 @@
 
 #include "runtime/url.h"
 
-#include "common/macos-ports.h"
-#include "runtime-common/stdlib/server/url-functions.h"
-#include "runtime-common/stdlib/string/string-functions.h"
-#include "runtime/array_functions.h"
-
 string AMPERSAND("&", 1);
 
-
-//string f$base64_decode (const string &s) {
-//  int result_len = (s.size() + 3) / 4 * 3;
-//  string res (result_len, false);
-//  result_len = base64_decode (s.c_str(), reinterpret_cast <unsigned char *> (res.buffer()), result_len + 1);
+// string f$base64_decode (const string &s) {
+//   int result_len = (s.size() + 3) / 4 * 3;
+//   string res (result_len, false);
+//   result_len = base64_decode (s.c_str(), reinterpret_cast <unsigned char *> (res.buffer()), result_len + 1);
 //
-//  if (result_len < 0) {
-//    return string();
-//  }
+//   if (result_len < 0) {
+//     return string();
+//   }
 //
-//  res.shrink (result_len);
-//  return res;
```

Comment:
extra changes

Reply:
Same as previous note. I don't want to change it in a PR with another goal.

Reply:
That's the point, but you do change these lines

Reply:
Sorry, misunderstanding. Fixed